lib/CL/devices/ventus/pocl_ventus.cc

```diff
@@ -660,11 +660,7 @@ uint64_t abuf_size = 0;
 	exit(1);
 #endif
 
-  ldssize=0x1000; //pass from elf file
```

Comment:
Code that repeatedly sets variables can be submitted separately. 
But why are the value settings of sgpr_usage and vgpr in the repeated code part different?

Reply:
hard code. it will be updated by compiler and pocl.